rainbow.go

```diff
@@ -68,3 +68,34 @@ func rgb(i float64) int {
 	blue := int(6*((math.Sin(freq*i+4*m)*127+128)/256)) * 1
 	return 16 + red + green + blue
 }
+
+// UnANSI color code syntax
+func UnANSI(str string) string {
```

Comment:
My Opinion, This is a temporary solution.
Each char have to structure of split text and color.